src/components/wishes/WishListComp.tsx

```diff
@@ -0,0 +1,97 @@
+import { DynamicData } from '../../@types/DynamicData';
+import Button from '../buttons/Button';
+import { RiDeleteBinFill } from 'react-icons/ri';
+import { motion } from 'framer-motion';
+import DeleteModal from '../Deletemodal';
+import ProductPageAddToCart from '../carts/ProductPageAddToCart';
+
+interface wishType {
+	wishes: DynamicData;
+	deleteModal: string;
```

Comment:
I don't think this deleteModal is string, maybe it's type should be ReaNode, check well!

Reply:
@leandreAlly here deleteModal must be string , i guess the issue is  name conversion but am going to rename